### ISSUE TITLE
Time and place

### ISSUE THREAD

**Issue by ijlyttle** [uncoast-unconf/uu-2019-day-zero#2]
I presume we will be at the main venue, John Deere?

What will be the hours for day-zero? 

cc @sctyner

**Comment by sctyner** [comment]
Yep, we have the John Deere venue for all 3 days. Hours are up to us, but I was thinking 10a-5p and then dinner with the larger group somewhere else after that.
